sql/parser/builtins.go

```diff
@@ -83,15 +86,15 @@ func (b builtin) match(types typeList) bool {
 // functions.
 var builtins = map[string][]builtin{
 	// TODO(XisiHuang): support encoding, i.e., length(str, encoding).
```

Comment:
@petermattis Yeah, UTF-8 is dominant these days. I guess the rare circumstance to not choose UTF-8 is to get optimal performance and  disk space usage for non english languages.